modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/wal/FileWriteAheadLogManager.java

```diff
@@ -869,7 +869,7 @@ private boolean hasIndex(long absIdx) {
     }
 
     /** {@inheritDoc} */
-    @Override public int truncate(WALPointer low, WALPointer high) {
+    @Override public int truncate(WALPointer low, WALPointer high, Collection<String> names) {
```

Comment:
@andrewmed 

All parameters in the same method in `FsyncModeFileWriteAheadLogManager` class were declared final.
Please do the same here.

Also please fix javadoc near `lastTruncatedArchiveIdx` field in these two classes: after changing signature it became incorrect.